pr2eus/make-pr2-model-file.l

```diff
@@ -1,11 +1,18 @@
-(defun make-pr2-model-file (&key (output-directory (ros::rospack-find "pr2eus")))
-  (let ((robot "pr2") s
-        fname_urdf fname_collada fname_yaml fname_lisp
-	valid-cameras camera-paths)
-    (unless (setq s (ros::get-param "/robot_description"))
-      (ros::ros-error "could not load model file from /robot_description~%")
-      (return-from make-pr2-model-file))
+;; re-define until https://github.com/jsk-ros-pkg/jsk_roseus/pull/629 is merged
+(defun ros::list-param ()
+  "Get the list of all the parameters in the server"
+  (let ((s (piped-fork "rosparam list")) tmp ret)
+    (while (setq tmp (read s nil))
+      (push (format nil "~A" tmp) ret))
+    ret))
+
+(defun get-pr2-names ()
+  ;; get list of rosparam
+  (reverse (mapcan #'(lambda (x) (if (substringp "robot_description" x) (list (subseq x 1 (- (length x) (length "/robot_description"))))))
```

Comment:
Thanks for updating, but error occurs while executing `make-pr2-model-file-test.launch`: https://travis-ci.org/jsk-ros-pkg/jsk_pr2eus/jobs/627167295#L5569-L5580

So we have to change that test launch file like
```
diff --git a/pr2eus/test/make-pr2-model-file-test.launch b/pr2eus/test/make-pr2-model-file-test.launch
index a43362d..0fa2292 100644
--- a/pr2eus/test/make-pr2-model-file-test.launch
+++ b/pr2eus/test/make-pr2-model-file-test.launch
@@ -1,9 +1,11 @@
 <launch>
-  <arg name="PR2_NO" default="1012" />
-  <rosparam command="load" file="$(find pr2eus)/test/pr$(arg PR2_NO)_urdf.yaml" param="robot_description"/>
-  <node name="pr2_camera_info" pkg="rosbag" type="play"
-       args="-l $(find pr2eus)/test/pr1012_camera_info.bag" />
+  <rosparam command="load" file="$(find pr2eus)/test/pr1012_urdf.yaml" param="/pr1012/robot_description"/>
+  <rosparam command="load" file="$(find pr2eus)/test/pr1040_urdf.yaml" param="/pr1040/robot_description"/>
+  <node name="pr1012_camera_info" pkg="rosbag" type="play"
+        args="--prefix=/pr1012 -l $(find pr2eus)/test/pr1012_camera_info.bag" />
+  <node name="pr1040_camera_info" pkg="rosbag" type="play"
+        args="--prefix=/pr1040 -l $(find pr2eus)/test/pr1040_camera_info.bag" />
   <test test-name="make_pr2_model_file_test" pkg="roseus" type="roseus"
-       args="$(find pr2eus)/test/make-pr2-model-file-test.l"
+        args="$(find pr2eus)/test/make-pr2-model-file-test.l"
         time-limit="1800" />
-</launch>
\ No newline at end of file
+</launch>
(END)
```

Reply:
- So we have to change that test launch file like
+ So we have to fix this error, to keep backward compatibility for example https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/426/commits/b891ba492c7c079099c3b32474de3b8e132cb291

and you fix did not test pr1040 case, so can you update the test code?